VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -74,7 +74,7 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
         provider: { VideoStreamViewController() }
     )
     
-    private var observer: PlayerObserver?
+    private var observer: SystemPlayerObserver?
```

Comment:
I forgot about observers of `Player`, ok it's not a big deal, i am just confusing about `System` part in name.